xml/hwbp_registers.xml

```diff
@@ -3,6 +3,36 @@
     prevent untrusted user code from causing entry into Debug Mode without the
     OS's permission.
 
+    \Rtdataone always contains a \Faction field, but the position varies
+    between trigger types. The encoding is, however, always the same and shown
+    in Table~\ref{tab:action}.
+
+    \begin{table}[H]
+    \centering
+    \caption{\Faction encoding}
+    \label{tab:action}
+    \begin{tabular}{|r|L|}
+    \hline
+    Value &amp; Description \\
+    \hline
+    0 &amp; Raise a breakpoint exception. (Used when software wants to use the
+    trigger module without an external debugger attached.) \\
+    \hline
+    1 &amp; Enter Debug Mode. (Only supported when \Fdmode is 1.) \\
```

Comment:
does the caveat of \Fdmode make sense outside of the context of this table?

Reply:
I don't quite understand your question, but I've clarified the language a little.